<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administrative Section</string>
  </property>
  <property name="windowIcon">
   <iconset resource="LaboratoryAlgorithm.qrc">
    <normaloff>:/Images/SurlyDexter.png</normaloff>:/Images/SurlyDexter.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="PatientTableView" name="patientTableView">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="addPatientPushButton">
            <property name="text">
             <string>&amp;Add Patient</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editPatientPushButton">
            <property name="text">
             <string>&amp;Edit Patient</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deletePatientPushButton">
            <property name="text">
             <string>&amp;Delete Patient</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="verificationPushButton">
            <property name="text">
             <string>Veri&amp;fication</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="prognosisPushButton">
            <property name="text">
             <string>Pro&amp;gnosis</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTableWidget" name="expertiseTableWidget">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="columnCount">
            <number>2</number>
           </property>
           <column/>
           <column/>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="viewExpertisePushButton">
             <property name="text">
              <string>&amp;View Expertise</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteExpertisePushButton">
             <property name="text">
              <string>De&amp;lete Expertise</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="conclusionGroupBox">
           <property name="title">
            <string>Conclusion</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTextEdit" name="conclusionTextEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="preeditPushButton">
             <property name="text">
              <string>P&amp;reedit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="printPushButton">
             <property name="text">
              <string>&amp;Print</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1036</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PatientTableView</class>
   <extends>QTableView</extends>
   <header>TableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="LaboratoryAlgorithm.qrc"/>
 </resources>
 <connections/>
</ui>
